driver/InputFlags.cpp

```diff
@@ -117,21 +118,20 @@ void InputFlags::Parse(int argc, char* argv[])
             printf("Illegal input flag\n");
             Print();
         }
-        else if(temp[0] == '-' && temp[1] == '-') // Long Name Input
+        else if(temp[0] == '-' && temp[1] == '-') // I/O/F layout long Input
         {
             std::string long_name = temp.substr(2);
             if(long_name == "help")
                 Print();
 
-            char short_name = FindShortName(long_name);
-
-            MapInputs[short_name].value = args[i + 1];
+            char layout_short_name = FindShortName(long_name);
+            UpdateLayoutValue(layout_short_name,args[i+1]);
```

Comment:
@atamazov Can you suggest a name? Can we rename the function to UpdateOptionalParameterValue, provided length of name is ok. Let me knw.

Reply:
Maybe "StoreOptionValue", because it stores the value of an option in the map? Your proposals also look acceptable. The point is that the implementation is not layout-specific but generic, so "layout" shouldn't be used in object names.

Reply:
Fixed in #1379